sabnzbd/filesystem.py

```diff
@@ -186,8 +186,11 @@ def has_win_device(filename: str) -> bool:
 
 CH_ILLEGAL = "/"
 CH_LEGAL = "+"
-CH_ILLEGAL_WIN = '\\/<>?*|"\t:'
-CH_LEGAL_WIN = "++{}!@#'+-"
+CH_ILLEGAL_WIN = '\\/<>?*|":'
+CH_LEGAL_WIN = "++{}!@#'-"
+for i in range(2, 32):
```

Comment:
Should this not start at 1 or 0?

Reply:
I thought it didn't work with those values but it turns out it was a different cause.